### PR TITLE
Manifest work template validation

### PR DESCRIPTION
**Pull request by jc-rh**

### CODE REVIEW

File: controllers/utils/manifestwork.go [hunk 1]
```diff
@@ -119,6 +119,16 @@ func GetManifestWorkForCluster(ctx context.Context, client client.Client, cluste
 	return mw, err
 }
 
+// GetManifestsFromTemplate returns the list of manifests from the manifestwork template
+func GetManifestsFromTemplate(ctx context.Context, client client.Client, name types.NamespacedName) (manifests []mwv1.Manifest, err error) {
```
Comment (sudomakeinstall2) [inline controllers/utils/manifestwork.go:123]: I think a unit test would have been good enough for this function. Kuttl test for this seems like a overkill to me.

**Review by sudomakeinstall2 — COMMENTED** [review]
/lgtm

File: controllers/manifestwork.go [hunk 2]
```diff
@@ -7,13 +7,28 @@ import (
 	utils "github.com/openshift-kni/cluster-group-upgrades-operator/controllers/utils"
 	ranv1alpha1 "github.com/openshift-kni/cluster-group-upgrades-operator/pkg/api/clustergroupupgrades/v1alpha1"
 	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
 	mwv1 "open-cluster-management.io/api/work/v1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 func (r *ClusterGroupUpgradeReconciler) validateManifestWorkTemplates(
-	ctx context.Context, clusterGroupUpgrade *ranv1alpha1.ClusterGroupUpgrade) (bool, []mwv1.Manifest, error) {
```
Comment (sudomakeinstall2) [inline controllers/manifestwork.go:15]: I think only a unittest is good enough here. Kuttl test seems a bit too much.

**Review by Missxiaoguo — APPROVED** [review]

**Pull request merged**